scripts/test-template.sh

```
@@ -30,17 +30,18 @@ TEST_APP_DIR="$PROJECT_ROOT/../test-app"
 # Parse arguments
 TEMPLATE_NAME="$1"
 VERSION="${2:-}"
+PACKAGE_MANAGER="${3:-yarn}"
 
 # Show usage if no template provided
 if [ -z "$TEMPLATE_NAME" ]; then
-    log_error "Usage: $0 <template-name> [version/branch-name]"
+    log_error "Usage: $0 <template-name> [version/branch-name] [package-manager]"
     echo ""
     echo "Available templates:"
     ls -1 "$TEMPLATES_DIR" | sed 's/^/  - /'
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)